WriteVHDLSyntax.py

```diff
@@ -563,8 +587,10 @@ def writeTopLevelMemoryType(mtypeB, memList, memInfo, extraports, delay = 0, spl
         # Write parameters
         parameterlist += "        RAM_WIDTH       => "+bitwidth+",\n"
         parameterlist += "        NUM_PAGES       => "+str(num_pages)+",\n"
-        if "MPROJ" in mem:
+        if "MPROJ" in mem :
             parameterlist += "        PAGE_LENGTH       => 64,\n"
+        if "MPAR" in mem:
+            parameterlist += "        PAGE_LENGTH       => 128,\n"
```

Comment:
@aryd Same comment as above. I think this parameter should just omitted:
```suggestion
```

Reply:
Removed.